ofl/newsreader/METADATA.pb

```diff
@@ -0,0 +1,37 @@
+name: "Newsreader"
+designer: "Production Type"
+license: "OFL"
+category: "SERIF"
+date_added: "2020-07-01"
+fonts {
+  name: "Newsreader"
+  style: "normal"
+  weight: 400
+  filename: "Newsreader[opsz,wght].ttf"
+  post_script_name: "Newsreader-Regular"
+  full_name: "Newsreader Regular"
+  copyright: "Copyright 2020 The Newsreader Project Authors (http://github.com/productiontype/Newsreader)"
+}
+fonts {
+  name: "Newsreader"
+  style: "italic"
+  weight: 400
+  filename: "Newsreader-Italic[opsz,wght].ttf"
+  post_script_name: "Newsreader-Italic"
+  full_name: "Newsreader Italic"
+  copyright: "Copyright 2020 The Newsreader Project Authors (http://github.com/productiontype/Newsreader)"
+}
+subsets: "latin"
+subsets: "latin-ext"
+subsets: "menu"
+subsets: "vietnamese"
+axes {
+  tag: "opsz"
+  min_value: 0.0
```

Comment:
This value should not have been accepted, and has been set to the axis minimum here

https://github.com/google/fonts/commit/92ec7cb6699bcdc28a8c23a38805820a2161cd66